listenbrainz_spark/mlhd/scripts/artist_popularity.py

```diff
@@ -0,0 +1,30 @@
+import os
+import listenbrainz_spark
+from datetime import datetime
+
+from listenbrainz_spark import config
+from listenbrainz_spark.mlhd import MLHD_DATA_PATH
+
+
+def main():
```

Comment:
Overall there are exception handling missing and we may want to choose a different data format for delivering the data (i.e. for inserting into and keeping a table inside a new schema inside the MB database). But we're not certain that this data will be useful, so we let's not delve into that right now.